sdk/objc/api/peerconnection/RTCPeerConnectionFactory.mm

```diff
@@ -260,7 +263,9 @@ - (instancetype)initWithNativeAudioEncoderFactory:
 - (RTC_OBJC_TYPE(RTCVideoTrack) *)videoTrackFromNativeTrack:(void *)nativeTrack {
   webrtc::MediaStreamTrackInterface *track = (webrtc::MediaStreamTrackInterface *)nativeTrack;
 
-  return [[RTC_OBJC_TYPE(RTCVideoTrack) alloc] initWithFactory:self nativeTrack:track];
+  return [[RTC_OBJC_TYPE(RTCVideoTrack) alloc] initWithFactory:self
```

Comment:
We may want to add the "RingRTC changes ..." comment?

Reply:
It's just out of sight above the method declaration. :-)